docs/designingProcesses/BasicNodes.md

```diff
@@ -1,6 +1,14 @@
+# Introduction
+
+Node works with a data stream. It can produce, fetch, send, collect data or organize data flow. Each node has at least two parameters: `Name` and `Description`. Name has to be unique in a scenario. Description is a narrative of your choice.  
+
+With the exception of Source and Sink nodes (link) all other nodes have input and at least one output flow.
```

Comment:
this is not quite true, some custom transformers and processors (not available in generic distribution) can end scenario. Maybe sth like "most of the nodes (with the exception of e.g. Source and Sink)"...